### PR TITLE
Add typings for graphql-list-fields

### PR DESCRIPTION
**Pull request by filipows**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/graphql-list-fields/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Type definitions for graphql-list-fields 2.0
+// Project: https://github.com/jakepusateri/graphql-list-fields#readme
+// Definitions by: Chris Filipowski <https://github.com/filipows>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { GraphQLResolveInfo } from "graphql";
+
+declare namespace getFieldList {}
```
Comment (plantain-00) [inline types/graphql-list-fields/index.d.ts]: It seems the empty namespace can be removed.

**Review by plantain-00 — CHANGES_REQUESTED** [review]

File: types/graphql-list-fields/index.d.ts (hunk 1 above)
Comment (filipows) [reply types/graphql-list-fields/index.d.ts]: I added this line in order to allow importing module with `import * as syntax`. For ex:
`import * as getFieldNames from 'graphql-list-fields';`

Comment (plantain-00) [reply types/graphql-list-fields/index.d.ts]: The empty namespace is a hack, and discouraged: https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports

Comment (filipows) [reply types/graphql-list-fields/index.d.ts]: Cheers Mate! You're right. I knew it's a hack, but I found 100 packages in DefinitelyTyped repo, that use empty namespaces, that's why I thought that maybe it's accepted. Apparently, as I see now, I was wrong and it actually shouldn't work as it's against the es6 standard. I updated the PR.

**Review by plantain-00 — APPROVED** [review]

**Pull request merged**